min_x: 480
max_x: 780
max_y: 360
min_prob: 0.30